packages/fuelet_secure_layer/lib/src/features/account/repository/accounts_local_repository_impl.dart

```diff
@@ -123,4 +124,13 @@ class AccountsLocalRepositoryImpl implements IAccountsLocalRepository {
 
     return updatedAccount;
   }
+
+  Account _changeWalletGroupIfNeeded(Account account) {
+    var updatedAccount = account;
+    if (account.walletGroup == WalletGroup.watchlist) {
+      updatedAccount = account.copyWith(walletGroup: WalletGroup.myWallets);
+      updateAccount(account);
+    }
```

Comment:
Shouldn't it be vise-versa? Should we set `watchlist` to all of the accounts?

Reply:
* if the account is not owned by the user

Reply:
we replace watchList to myList, and if user have a account in watchList group, we rewrite account to myWatchList

Reply:
It states [here](https://fuelet.atlassian.net/browse/FUELET-915) that we should set wallet group to `watchlist`, not `myWallets`

Reply:
fixed